compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -487,9 +488,9 @@ fn struct_def_fields(
 
     let mut fields = im::Vector::new();
 
```

Comment:
Should `struct_def_fields` / `StructDefinition::fields` also return the visibilities?

Reply:
Same as above: I didn't want to change the macros API in this PR.